src/components/chips/js/chipsDirective.js

```diff
@@ -62,6 +62,8 @@
    *  </span>
    *
    * @param {string=|object=} ng-model A model to bind the list of items to
+   * @param {expression=} md-on-chip-select An expression to be run each time a new item got selected
```

Comment:
Why rename this `md-on-chip-select` instead of just `md-on-select`? Is the thought that we may want to know when the chips container is selected/deselected?

Normally, I wouldn't really even mention this, but this is changing a public API very close to our 1.0 release, and I'm hesitant to do so in an RC.